### ISSUE TITLE
Strange code: blck_0 = MAX(32, 32)

### ISSUE THREAD

**Issue by GermanAizek** [ggerganov/llama.cpp#5506]
https://github.com/ggerganov/llama.cpp/blob/0d4177126b0556e202efb85bf3f768be81076400/ggml.c#L10580-L10582

@ggerganov,
`blck_0` have `MAX(32, 32)` is it exactly correctly defined? then why call define `MAX` at all?

**Comment by ggerganov** [comment]
Yes, it is ok because developers might want to experiment with different values for `GGML_VEC_MAD_UNROLL`
